app/styles/app-dark.css

```diff
@@ -85,6 +85,19 @@ input,
     background: color-mod(var(--lightgrey));
 }
 
+input:focus,
+.gh-input:focus,
+.gh-select:focus,
+.gh-select select:focus {
+    border-color: color-mod(var(--darkgrey) l(-27%) blackness(15%));
+}
+
+.for-checkbox label:hover input:not(:checked) + .input-toggle-component,
+.for-radio label:hover input:not(:checked) + .input-toggle-component {
+    /* light=>dark theme => lightgrey=>middarkgrey */
+    border-color: color-mod(var(--middarkgrey) l(-15%) s(-10%));
```

Comment:
Same here as for the previous one: 
`border-color: color-mod(var(--lightgrey) l(+10%));`